inbm/Changelog.md

```diff
@@ -14,6 +14,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Added
  - HSD 15011298299, RTC 515263 - Support non-LTS versions of Ubuntu
+ - RTC 496923 - Return status is 302 when preinstall check fail
```

Comment:
This message doesn't really tell me what the change is.  It should be something more like.  'Changed return status during preinstall check fail from 302 to 400'.

Reply:
Updated as per suggestion --> RTC 496923 - Changed return status during preinstall check fail from 302 to 400 and updated the error message